notebooks/prometheus2_evaluation.ipynb

```diff
@@ -0,0 +1,12347 @@
+{
```

Comment:
<p>nit: a newest release will already show the column <code>metrics</code>, and not that columns as the index in the dataframe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>OK for now let's leave it as is.</p>